scout/dashboard.md

```diff
@@ -121,7 +120,9 @@ Each entry in the list shows the following details:
 - The versions of the package used by images in the organization.
 - The number of images that use the package.
 
-### Base images
+## Base images
+
+The **Base images** tab shows all base images images are based upon in an organization.
```

Comment:
**nitpick:** "base images images are based upon" sounds like a mouthful 😬 

**suggestion:** 
> The **Base images** tab shows all base images used in an organization.

In my opinion, I could be wrong, the term `base image` is pretty much self-explanatory so we shouldn't have to extend the sentence

Reply:
@fdiogo It does sound like a mouthful, but I am not sure it is completely clear as a concept… I guess it is really.

```suggestion
The **Base images** tab shows all base images images used by repositories in an organization.
```

How about that?

Reply:
We still have the double `images` but it is better 👍

Reply:
Yeah, I know :( Let's use it for now.